Description: Disable the code signing with a self signed certificated.
 This doesn't play well with the manifest modification performed by
 jh_manifest to make the jar executable (and signing with a self signed
 certificate doesn't provide any security anyway).
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/janino-dist/build.xml
+++ b/janino-dist/build.xml
@@ -44,13 +44,6 @@
 			</manifest>
 			<fileset dir="build/bin-commons-compiler"/>
 		</jar>
-		<signjar
-			keystore="dummy-keystore"
-			storepass="storepass"
-			keypass="keypass"
-			jar="build/commons-compiler.jar"
-			alias="dummy"
-		/>
 
 		<!-- Build 'build/janino.jar'. -->
 		<delete dir="build/bin-janino"/>
@@ -79,13 +72,6 @@
 			</manifest>
 			<fileset dir="build/bin-janino"/>
 		</jar>
-		<signjar
-			keystore="dummy-keystore"
-			storepass="storepass"
-			keypass="keypass"
-			jar="build/janino.jar"
-			alias="dummy"
-		/>
 
 		<!-- Build 'build/commons-compiler-jdk.jar'. -->
 		<delete dir="build/bin-commons-compiler-jdk"/>
@@ -114,13 +100,6 @@
 			</manifest>
 			<fileset dir="build/bin-commons-compiler-jdk"/>
 		</jar>
-		<signjar
-			keystore="dummy-keystore"
-			storepass="storepass"
-			keypass="keypass"
-			jar="build/commons-compiler-jdk.jar"
-			alias="dummy"
-		/>
 
 		<!-- Build source ZIP files. -->
 		<zip destfile="build/commons-compiler-src.zip" filesonly="true">
